dataminer-overview/Pricing/Pricing_Perpetual_Use_Licensing.md

```diff
@@ -44,10 +46,18 @@ In addition to the [standard functions](https://community.dataminer.services/cor
 | DataMiner Object Models (DOM) | - | - | x |
 
 *Included (x)*
-
 > [!NOTE]
 > Tier-1 is included by default with volume-based licenses, i.e. System Volume Equipment and Experience and Performance Management.
 
+
+Please note that while some additional functions may be accessible without a license, full functionality may only be achieved when paired with a licensed function. For example, DataMiner Infrastructure Discovery Provisioning ([IDP](https://community.dataminer.services/dataminer-apps/dataminer-infrastructure-discovery-and-provisioning-app/)) requires Automation as a minimum, with Process Automation being highly recommended.
+
+
+> [!WARNING]
```

Comment:
Same comment as above: should this definitely be a warning?

Reply:
this seems to be a warning, if people try to use IDP without having automation, it will not work